lib/FFI/Platypus/Buffer.pm

```diff
@@ -127,6 +128,34 @@ sub buffer_to_scalar ($$)
 
 1;
 
+=head2 grow
+
+ grow $scalar, $size;
+ grow $scalar, $size, $clear;
+
+Ensure that the scalar can contain at least C<$size> bytes.  C<$clear>
+determines if the scalar is cleared before it is enlarged, which
+avoids copying the original contents to newly allocated storage.  If
+C<$clear> is not specified or is true, the scalar is cleared,
+otherwise it is not.  For example, after
+
+   $scalar = "my string";
+   grow $scalar, 100, 0;
+
+C<$scalar == "my string">, while after
+
+   $scalar = "my string";
+   grow $scalar, 100;
+
+C<length($scalar) == 0>
+
+Any pointers obtained with C<scalar_to_pointer> or C<scalar_to_buffer>
+are no longer valid after growing the scalar.  By default,
+
+
+=cut
+
+
```

Comment:
nit: lines 154-158 should be replace by a single blank line.   (single blank line between content and POD directives, and no need to `=cut` when the next thing is a `=head1`.